public/js/showoff.js

```diff
@@ -1689,7 +1689,7 @@ function setupStats(data)
 
 /* Is this a mobile device? */
 function mobile() {
-  return ( $(window).width() <= 480 )
+  return ( $(window).width() <= 640 )
```

Comment:
The point at which nothing fits reasonably into the top bar.  That's what the media query came out to be, so I decided to update the function to match